src/main/java/app/LearningOutcome.java

```diff
@@ -27,12 +38,15 @@ public LearningOutcome(String name, String description) {
     }
 
     /**
-     * Constructor incase name not provided. Gives it a blank name but retains description
-     * @param description
+     * Constructor for learning outcomes
+     * @param name - Name for learning outcomes, to easily identify
+     * @param description - Description used to display what the learning outcome is
+     * @param prog - program this learning outcome belongs to
```

Comment:
Learning outcomes should belong to courses which in turn belong to programs. Do we extend this relationship to the one I mentioned in the next iteration?

Reply:
Well the first line of the last parah describing the project says "A user should be able to list, for a given year and for a given program: all the learning outcomes," so I assumed we should make it easy to establish that direct relationship. Or do I have this wrong? A user should be able to pick the year and program and see all the learning outcomes if he chooses that filter.

Reply:
You might be right though. Once we have the courses in place, this could be fixed.